src/SpaceKitProvider/index.tsx

```diff
@@ -47,12 +47,25 @@ const SpaceKitSetContext = React.createContext<
  */
 export const SpaceKitProvider: React.FC<Partial<State>> = ({
```

Comment:
Ooh could we pick the values, so no one can pass the singleton prop by accident :)
```suggestion
export const SpaceKitProvider: React.FC<Partial<Pick<State, "theme" | "disableAnimations">>> = ({
```

Reply:
oh right! 🙏🏽